api/utils/keys/yubikey_common.go

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright 2022 Gravitational, Inc.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package keys
+
+import "context"
+
+func GetOrGenerateYubiKeyPrivateKey(ctx context.Context, touchRequired bool) (*PrivateKey, error) {
+	return getOrGenerateYubiKeyPrivateKey(ctx, touchRequired)
```

Comment:
So that the global function `GetOrGenerateYubiKeyPrivateKey` would be non-build dependent, while the local function `getOrGenerateYubiKeyPrivateKey` would change depending on `libpcsclite`, in `yubikey.go` or `yubikey_other.go`. 

I copied this idea from the libfido2 implementation.